MediaBrowser.Model/Dlna/ContentFeatureBuilder.cs

```diff
@@ -151,10 +151,10 @@ public class ContentFeatureBuilder
                             DlnaFlags.InteractiveTransferMode |
                             DlnaFlags.DlnaV15;
 
-            // if (isDirectStream)
-            // {
-            //     flagValue = flagValue | DlnaFlags.ByteBasedSeek;
-            // }
+            if (isDirectStream)
+            {
+                flagValue = flagValue | DlnaFlags.ByteBasedSeek;
+            }
             //  else if (runtimeTicks.HasValue)
             // {
             //     flagValue = flagValue | DlnaFlags.TimeBasedSeek;
```

Comment:
What about this one?

Reply:
From what I know runtimeTicks.HasValue applies to transcoded streams where byte based seeking isn't available. 
Currently I wasn't able to get seek to work when transcoding on this TV and adding the DLNA.ORG_OP to a transcoded stream actually breaks playback with "Format not supported" error message on the TV.
On a Samsung Q6FNA TV with Tizen the changes have no effect and seek was working no matter what.

Reply:
Ok, can you add that as a comment so we know why TimeBasedSeek is disabled?

Reply:
I have added comments regarding why time seek is still disabled during transcoding.